app/src/main/java/org/oppia/android/app/options/OptionsActivityPresenter.kt

```diff
@@ -109,11 +109,11 @@ class OptionsActivityPresenter @Inject constructor(
   }
 
   fun loadAudioLanguageFragment(audioLanguage: String) {
-    val defaultAudioFragment =
-      DefaultAudioFragment.newInstance(AUDIO_LANGUAGE, audioLanguage)
+    val audioLanguageFragment =
+      AudioLanguageFragment.newInstance(AUDIO_LANGUAGE, audioLanguage)
     activity.supportFragmentManager
       .beginTransaction()
-      .add(R.id.multipane_options_container, defaultAudioFragment)
+      .add(R.id.multipane_options_container, audioLanguageFragment)
       .commitNow()
     getOptionFragment()?.setSelectedFragment(DEFAULT_AUDIO_FRAGMENT)
```

Comment:
change `DEFAULT_AUDIO_FRAGMENT` to `AUDIO_LANGUAGE_FRAGMENT`

Reply:
Done